frontend/src/canvas/CanvasContainer.js

```diff
@@ -158,12 +169,26 @@ const CanvasContainer = (props) => {
       return;
     }
 
+    // Erase Extra Pixel
+    if (props.isEraserMode) {
+      const pixel = props.extraPixelsData.filter((pixelData) => {
+        return pixelData.x === x && pixelData.y === y;
+      });
+      const pixelIndex = props.extraPixelsData.indexOf(pixel[0]);
```

Comment:
I think using `findIndex` instead of `filter` then `indexOf` would be best for this